tensorflow_probability/python/mcmc/sample_test.py

```diff
@@ -309,6 +334,29 @@ def log_prob(x):
     # Checking that shape inference doesn't fail.
     sample(2)
 
+  def testSeedReproducibility(self):
+    first_fake_kernel = RandomTransitionKernel()
+    second_fake_kernel = RandomTransitionKernel()
+    seed = samplers.sanitize_seed(test_util.test_seed())
+    for num_results in range(2, 5):
```

Comment:
If you really feel this loop is necessary, then I'd recommend using parameterized testing for this. The reason being is that if this test were to fail as it is now, you'd have no idea what `num_results` was when it failed (you could log it, certainly, but the idiomatic way to solve this issue is to use parameterized tests).

Reply:
good point! The loop is probably actually not that necessary.